views/recent.ejs

```diff
@@ -1,26 +1,25 @@
 <% include ./partials/header %>
-<div class="resultContainer col-xs-12">
-  <h1>Recently scanned QR code</h1>
-  <article class="col-xs-12">
-    <ul class='recent-list'>
-      <% for(var i = 0; i <allScans.length; i++){%>
-        <% if(allScans[i].fixture) { %>
-          <li>
-            <a href="/<%- allScans[i].uuid %>"><%- allScans[i].time.toTimeString() %> - <%- allScans[i].fixture %> </a> :<br/>
-            <%- allScans[i].uuid %>
-          </li>
+<h1>Recently scanned QR codes</h1>
+<article class="col-xs-12">
+  <ul class='recent-list'>
+    <% for (var i = 0; i < allScans.length; i++) { %>
+      <li>
+        <%- allScans[i].time.toTimeString() %> -
+        <% if (allScans[i].fixture) { %>
+          <%- allScans[i].fixture %>
         <% } else { %>
-          <li>
-            <a href="/<%- allScans[i].uuid %>"><%- allScans[i].time.toTimeString() %>
-             </a>
-            <span style="color:<%- allScans[i].status==='fixed'?'orange':'red'%>" >
-              :<br/>
-              <%- allScans[i].uuid %>
-            </span>
-          </li>
-        <%}%>
-      <%}%>
-    </ul>
-  </article>
-</div>
+          ???
+        <% } %>
+        : <br/>
+        <a href="/<%- allScans[i].uuid %>"><%- allScans[i].uuid %></a>
+        <% if (!allScans[i].fixture) { %>
+          <a href="<%- allScans[i].link %>" style="color: <%- allScans[i].status === 'fixed' ? 'orange' : 'red' %>">Missing Details!</a>
```

Comment:
it makes sense to edit style inline here because all we want is to change a color.
but it makes the job of a designer who only knows css more difficult.
consider just giving a style class (class=fixed or class=broken) and edit that style in the css file